### DISCUSSION TITLE
Ping favorite list

### DISCUSSION THREAD

**Discussion by ZloNik** [isontheline/pro.webssh.net#873] [Ideas]
Greetings! I periodically check the ping of servers through third-party applications, as it implements the preservation of frequently used addresses. 
What is the probability of such an option? 
Here is one of the successful examples of such an implementation: https://clck.ru/34EBoo

**Comment by isontheline** [comment]
Hello @ZloNik 👋

Thank you a lot for reaching me out 🙏

Yes a such feature could be implemented inside WebSSH.
However I can’t give you any ETA as I’m late on other issues 😓

Do you need to ping only on servers you already have put inside « SSH / SFTP » sections?
Or you need a custom « ping list » with hosts not inside SSH / SFTP?

**Reply by ZloNik** [reply]
It will be useful to add any server address, both incoming and non-incoming to the SSH / SFTP section

**Reply by isontheline** [reply]
Thanks for your reply!
I’ve opened an issue to track this feature : https://github.com/isontheline/pro.webssh.net/issues/874
